pandas/tests/reductions/test_reductions.py

```diff
@@ -856,6 +856,16 @@ def test_idxmax(self):
         result = s.idxmin()
         assert result == 1.1
 
+        # GH#36566
```

Comment:
make this a separate test, pls parameterize over the input args as much as possible